content/docs/tutorials/aws/aws-go-webserver.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Web Server Using Amazon EC2 (in Go) | Go"
```

Comment:
I prefer the consistency and then once this is merged in, we go back and update the titles in the tutorials. It could be better, in an SEO sense, to have the language in the title itself instead of appended to the end.

The goal of this change was to eliminate duplicate titles amongst our tutorials (which this achieves). Once our duplication problem is gone, we can look at optimizing the SEO for these tutorials.